recipes/rgi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "rgi" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "17abb2e89608b3d82a491d3b8e894456da6d3c955df6e1563ae9240b02b3d0d1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://card.mcmaster.ca/download/1/software-v{{ version }}.tar.gz
+  fn: rgi-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+
+build:
+  skip: True # [not py27]
+  number: 0
+
+requirements:
+  build:
+    - biopython
+    - six >=1.7.0
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - zlib {{ CONDA_ZLIB }}*
+  run:
+    - filetype
+    - blast
+    - biopython
+    - six >=1.7.0
+    - libgcc    # [not osx]
+    - zlib {{ CONDA_ZLIB }}*
+    - prodigal
+    - diamond >=0.8.24.86
+    
+test:
+  commands:
+    - rgi load -h
+    - rgi main -h
+    - rgi tab -h
+    - rgi parser -h
+    - rgi clean -h
+    - rgi galaxy -h
+
+
+about:
+  home: https://card.mcmaster.ca
+  license: https://card.mcmaster.ca/about
```

Comment:
Please add
```yaml
  license_file: LICENSE
```
As for the license itself, it should definitely state that it's non-commercial. I didn't read https://card.mcmaster.ca/about, so can't give exact advices yet